style/_base.scss

```diff
@@ -1,35 +1,210 @@
+// }
 
 html {
-  box-sizing: border-box;
-
   font-size: 62.5%; // 1rem = 10px, 10px/16px = 62.5%
 
   // @media only screen and (max-width: $bp-large) {
   // font-size: 50%;
   // }
 }
 
-li {
-  margin-left: 0;
-  padding-left: 0;
-  list-style-type: none;
-}
+// li {
+//   margin-left: 0;
+//   padding-left: 0;
+//   list-style-type: none;
+// }
 // Header
 
 .header-box {
   max-width: 100%;
   background-color: $colorcopper;
+  margin: 0 auto;
+  padding: 0 3em 1.5em;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  .menu-box {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    align-items: center;
+    .logo {
+      font-size: 70px;
+      color: $colorwhite;
+    }
+  }
+  .navigation {
+    &-left {
+      &_list {
+        display: flex;
+        list-style-type: none;
+        &_elem {
+          justify-content: space-around;
+          margin-top: 0;
+          &_name {
+            color: $colorwhite;
+            font-size: 1.6em;
+            padding: 0.5em 1em;
+          }
+        }
+      }
+    }
+    &-right {
+      &_list {
+        display: flex;
+        padding: 0;
+        &_elem {
+          list-style: none;
+          margin-top: 0;
+          &_name {
+            color: $colorwhite;
+            font-size: 1.6em;
+            padding: 0.5em 1em;
+            display: flex;
+          }
+        }
+      }
+    }
+  }
+  .afte-header {
+    display: flex;
+    align-items: flex-start;
+    flex-direction: column;
+    padding-left: 150px;
+    padding-bottom: 59px;
+    &_title {
+      font-size: 6em;
+      line-height: 76px;
+      color: $colorwhite;
+      margin: 0;
+      &_text {
+        font-size: 1.9em;
+        letter-spacing: 0.04em;
+        color: $colorwhite;
+        margin: 10px 0 35px 0;
+      }
+    }
+    .button-btn {
+      height: 80px;
+      @include btn {
+        max-width: 350px;
+        background-color: $colorcopper;
+      }
+    }
+  }
+}
+.box-one {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  background-color: $coloryellow;
+  padding: 0 3em 1.5em;
+  &_main {
+    display: flex;
+    align-items: flex-start;
+    flex-direction: column;
+    padding-left: 150px;
+    padding-bottom: 59px;
+    &_title {
+      font-size: 6em;
+      line-height: 76px;
+      color: $colorwhite;
+      margin: 0;
+      &_text {
+        font-size: 1.9em;
+        letter-spacing: 0.04em;
+        color: $colorwhite;
+        margin: 10px 0 35px 0;
+      }
+    }
+    .button-btn {
+      height: 80px;
+      @include btn {
+        max-width: 350px;
+        background-color: $coloryellow;
+        &:hover {
+          background-color: red($color: #000000);
+        }
+      }
+    }
+  }
 }
-.menu-box {
-  max-width: 96vw;
+.box-two {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  &_main {
+    display: flex;
+    width: 100vw;
+    .block-one {
```

Comment:
never use names like "block 1", "block 2"  rename this for example content or info